zsh_history_to_fish/command.py

```diff
@@ -59,7 +59,7 @@ def writer_factory(output_file, dry_run):
 @click.command(help=__description__)
 @click.version_option()
 @click.argument('input_file', type=click.Path(exists=True), required=False,
-                default=os.path.expanduser(ZSH_HISTORY_FILE))
+                default=os.path.expanduser(HISTFILE))
```

Comment:
Did you mean to do this?
```suggestion
                default=os.getenv(HISTFILE))
```
HISTFILE is not a variable in this file, but an environment variable.

Reply:
Ah yes, that was what I meant. The dangers of drive-by PRs!